src/ai/backend/manager/scheduler/dispatcher.py

```diff
@@ -634,7 +634,7 @@ async def _pipe_builder(r: Redis) -> RedisPipeline:
         return [
             ag
             for ag, count in zip(candidate_agents, raw_counts)
-            if max_container_count > int(count)
+            if count is not None and max_container_count > int(count)
```

Comment:
Hm, I think we should include the agent when we cannot determine the current container count of it, as a fallback. Let's treat `None` as `0`.

Reply:
thanks, I updated the code!